packages/mask/src/plugins/Collectible/pipes/index.ts

```diff
@@ -29,8 +29,8 @@ export const resolveOpenSeaNetwork = createLookupTableResolver<ChainId.Mainnet |
 
 export const resolveCollectibleProviderName = createLookupTableResolver<SourceType, string>(
```

Comment:
Remove this resolver. Use `resolveSourceName` as replacement.

Reply:
done